cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -308,7 +291,13 @@ def upload_rna_coverage_bigwig_to_scout(self, case_id: str, dry_run: bool) -> No
                 is_tumour=rna_sample.is_tumour,
             )
 
-            self._check_cases_single_connected(dna_cases, rna_sample)
+            upload_dna_cases.update(dna_cases)
+
+            if not upload_dna_cases:
```

Comment:
Since the same check is made three times on a similar list can we make this a function that takes a list and a file type (eg. rna_coverage_bigwig) and then just call that in the three functions?

Reply:
Agreed, will add that in future commit.

Reply:
A new function is now added, create_rna_dna_mapper, which creates a dictionary with the relationships between rna and dna samples. This can then be called by the 3 rna upload functions.